.github/workflows/ci-test.yml

```diff
@@ -989,3 +989,94 @@ jobs:
           # or python docs `-p` on CI checks until we resolve the best way to
           # build them for deployment
           ./scripts/build_docs.sh -b
+
+  land-blocking-test-forge:
+    name: Run land blocking test in Forge
+    runs-on: self-hosted
+    needs: [prepare, build-images, need-base-images]
+    if: ${{ always() && needs.build-images.result=='success' && needs.prepare.outputs.test-land-blocking == 'true' }}
+    outputs:
+      lbt-result: ${{ steps.post-test-result.outputs.forge-result }}
+    timeout-minutes: 45
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          # This ensures that the tip of the PR is checked out instead of the merge between the base ref and the tip
+          # On `push` this value will be empty and will "do-the-right-thing"
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Launch forge test
+        # NOTE Remember to update PR comment payload if cti cmd is updated.
+        run: |
+          set +e
+          date
+          export FGI_OUTPUT_LOG=$(mktemp)
+          echo "FGI_OUTPUT_LOG=$FGI_OUTPUT_LOG" >> $GITHUB_ENV
+          cmd=""
+          if ${{ needs.need-base-images.result!='success' }}; then
+            cmd="./scripts/fgi/run --tag ${{ needs.build-images.outputs.head-tag }} --suite land_blocking"
+          else
+            cmd="./scripts/fgi/run -E SLACK_URL=${{ secrets.WEBHOOK_FORGE }} --tag ${{ needs.build-images.outputs.head-tag }} --base-image-tag ${{ needs.need-base-images.outputs.prev-tag }} --suite land_blocking_compat"
```

Comment:
Did you only intend for `SLACK_URL` to be set on one of these code paths?

Reply:
there is another place in compare_lbt_signals job, but good to know if there has any other fancy way to set it?